dashboard/src/deployment/DeploymentRow.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { Icon, Table } from 'semantic-ui-react';
+//import logo from './logo.svg';
+//import './App.css';
+
+class DeploymentRow extends Component {
+  render() {
+    return (
+      <Table.Row>
+        <Table.Cell><Icon name="bell" color="red"/></Table.Cell>
+        <Table.Cell>{this.props.info.name}</Table.Cell>
```

Comment:
No reason to use a class based component unless you have some kind of state. Also I'd unpack the info properties into props directly.